weaver/core/drivers/corda-driver/jre8.Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian:12
```

Comment:
Is this image being used anywhere? If so, perhaps, we should add a `Makefile` target.

Reply:
Sorry no, some files have been added by mistake it seems, I'll remove them.